sources/packages/forms/src/form-definitions/confirmsstudentenrollment.json

```diff
@@ -412,6 +412,595 @@
           "input": false,
           "tableView": false,
           "components": [
+            {
+              "label": "HTML",
```

Comment:
Outside the PR context. During the grooming, we decided to keep converting the COE page from form.io to Vuetify outside the parttime release scope but still have a ticket created for the same to not lose track of it.